### DISCUSSION TITLE
book/kubernetes-handbook/objects/pod-overview/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#350] [Announcements]
# book/kubernetes-handbook/objects/pod-overview/

宋净超的博客

https://jimmysong.io/book/kubernetes-handbook/objects/pod-overview/

**Comment by SOSACTWT** [comment]
[Pod 中如何管理多个容器](https://jimmysong.io/book/kubernetes-handbook/objects/pod-overview/#pod-%E4%B8%AD%E5%A6%82%E4%BD%95%E7%AE%A1%E7%90%86%E5%A4%9A%E4%B8%AA%E5%AE%B9%E5%99%A8)这一节的图好像没了。

**Reply by rootsongjc** [reply]
已修复，感谢反馈。

**Comment by ocean-Ghy** [comment]
怎么下载这本书啊

**Reply by rootsongjc** [reply]
因为本书章节和页面繁多，书中又有大量的跳转，现在使用 Hugo 来编译，暂时无导出 PDF 功能。因此本书最新版本只提供在线浏览，暂不提供下载。旧版可以在这里找到 https://github.com/rootsongjc/kubernetes-handbook

**Comment by socake** [comment]
感谢大佬~~~
